pkgs/applications/window-managers/sway/default.nix

```diff
@@ -47,16 +55,22 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    wayland libxkbcommon pcre json_c dbus libevdev
+    wayland libxkbcommon pcre json_c ]
+  ++ lib.optionals dbusSupport [ dbus ]
```

Comment:
This makes sense to keep this a refactoring but for better readability I'd prefer to add this at the end instead of splitting up the list.

Reply:
My instinct is generally to maintain the ordering of the buildInputs in the default case, to avoid changing the derivation-hash and therefore triggering unnecessary rebuilds for people who do not customize the value of `dbusSupport`.

I guess in this case it doesn't matter much since very few packages depend on `sway`.  I've implemented your recommendation and will push a new commit in a moment.